schemas/1.3.0/adaptive-card.json

```diff
@@ -982,19 +938,19 @@
         },
         "max": {
           "type": "string",
-          "description": "Hint of maximum value expressed in ISO-8601 format (may be ignored by some clients)."
+          "description": "Hint of maximum value expressed in YYYY-MM-DD(may be ignored by some clients)."
         },
         "min": {
           "type": "string",
-          "description": "Hint of minimum value expressed in ISO-8601 format (may be ignored by some clients)."
+          "description": "Hint of minimum value expressed in YYYY-MM-DD(may be ignored by some clients)."
```

Comment:
This should indeed be ISO8601. It has to support a time offset, otherwise it will not work properly across time zones.

Reply:
Should we remove "may be ignored by some clients"?